src/analyze/analyze-plot.c

```diff
@@ -158,12 +159,46 @@ static void svg_graph_box(double height, double begin, double end) {
                             SCALE_Y * height);
         }
 }
+
+static void plot_tooltip(const char *name, const RelatedUnits *r) {
```

Comment:
`r` is used as a special variable for storing return value of function call in our codebase. Please choose some other name for this.